tests/CleanCode.php

```diff
@@ -28,4 +28,10 @@ public function thisReadsPrivateClassMember(): string
     {
         return $this->privateClassMember;
     }
+
+    public function thisUsesAInternalFunction(): string
+    {
+        // Uses '\md5' instead of 'md5' to improve opcode handling
+        return \md5('hashMe');
```

Comment:
Ist der Test wirklich richtig?

Reply:
Yes, the 'test' only checks if the class passes the coding standards. In the future we should try to use it the other way around. But for the moment it is ok

Reply:
ah okay ...